test_rmw_implementation/test/test_serialize_deserialize.cpp

```diff
@@ -14,8 +14,12 @@
 
 #include <gtest/gtest.h>
 
+#include "osrf_testing_tools_cpp/scope_exit.hpp"
```

Comment:
It looks like we need an `ament_target_dependencies` on `osrf_testing_tools_cpp` for this test now.

Reply:
Indeed.

Reply:
Done